src/views/Menu/Dishes.tsx

```diff
@@ -0,0 +1,14 @@
+import { Heading, Link } from '@chakra-ui/react';
+import { Link as RouterLink } from 'react-router-dom';
+import { ROUTE_MENU_FORMS } from 'routing';
+
+const Dishes = () => (
```

Comment:
When I was implementing initial model I called it `recipes` - `receptury` (as in documentation).
We need to decide which one is better and stick to one nomenclature.

Reply:
I would stick with the "dishes" nomenclature.

Reply:
Fine, so please change names of interfaces and collections in Firebase respectively.